### PR TITLE
dependency update

### PR DESCRIPTION
**Pull request by thomasridd**
Various dependency updates by comparing to the typescript-template and renovate

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7
```
Comment (SimonMitchellMOJ) [inline .circleci/config.yml:4]: we don't want to pin these to specific versions just using 7 will use the latest version of 7.
Also there is hmpps@9 of the orb

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-  browser-tools: circleci/browser-tools@1.4
```
Comment (SimonMitchellMOJ) [inline .circleci/config.yml:5]: some we don't want to pin to the specific patch version for browser tools

File: .circleci/config.yml (hunk 1 above)
Comment (thomasridd) [reply .circleci/config.yml:4]: do we want to upgrade that to 9 then?

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7
+  hmpps: ministryofjustice/hmpps@9
   browser-tools: circleci/browser-tools@1.4
-  slack: circleci/slack@4.12
+  slack: circleci/slack@4.12.5
```
Comment (SimonMitchellMOJ) [inline .circleci/config.yml:6]: Probably just want to leave this as 4.12 as well

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**